frontend/src/pages/ShowTransactionLinkInformations.vue

```diff
@@ -1,34 +1,48 @@
 <template>
   <div class="show-transaction-link-informations">
-    <!-- Header -->
-    <div class="header py-7 py-lg-8 pt-lg-9">
-      <b-container>
-        <div class="header-body text-center mb-7">
-          <p class="h1">
-            {{ displaySetup.user.firstName }}
-            {{ $t('transaction-link.send_you') }} {{ displaySetup.amount | GDD }}
-          </p>
-          <p class="h4">{{ displaySetup.memo }}</p>
-          <hr />
-          <b-button v-if="displaySetup.linkTo" :to="displaySetup.linkTo">
-            {{ $t('transaction-link.button') }}
-          </b-button>
-        </div>
-      </b-container>
-    </div>
+    <div class="text-center"><b-img :src="img" fluid alt="logo"></b-img></div>
```

Comment:
Can we call this file `TransactionLink.vue`

Reply:
geändert